src/Proxies/WC.php

```diff
@@ -145,4 +145,15 @@ public function get_shipping_zones(): array {
 	public function get_shipping_zone( int $zone_id ): ?WC_Shipping_Zone {
 		return WC_Shipping_Zones::get_zone( $zone_id );
 	}
+
+	/**
+	 * Get Base Currency Code.
+	 *
+	 * @return string
+	 *
+	 * @since x.x.x
+	 */
+	public function get_woocommerce_currency(): string {
```

Comment:
Do we really need this in a proxy? I don't understand the design of the plugin too much so if this is how things are done then OK, otherwise, this feels like overkill.

Reply:
I agree that this file is getting bloated but I think we should still add these here because it allows us to mock these easily in our unit tests.